schemas/user.py

```diff
@@ -1,15 +1,15 @@
 from datetime import datetime
 from typing import List, Dict, Union
 import re
-
 from pydantic import BaseModel, validator
+from components.exceptions import BackendException
 
 
 class UsersListViewOut(BaseModel):
 
     user_id: int
     login: str
-    last_vist: datetime = None
+    last_vist: datetime = None      # TODO syntax error
```

Comment:
Change it to `last_visit` if this field has no syntax errors inside declaration of view (check main.sql, and ...views.py for that)

Reply:
ok